### ISSUE TITLE
Testplugin 1.0.225.1 crashes OpenCPN 5.7.1

### ISSUE THREAD

**Issue by rgleason** [jongough/testplugin_pi#286]
In windows 11 testplugin 1.0.225.1 crashes windows when you hit delete text point, etc.
for High res screens the menus are out of whack.
Plugin opens ok and can view the menus.
There is a first message of functions not available.

![Screenshot (849)](https://user-images.githubusercontent.com/2363959/214562520-f00c40e7-ceb7-49c7-bb91-f2eba3025e0a.png)
![Screenshot (850)](https://user-images.githubusercontent.com/2363959/214562538-4580bd25-cf03-4c53-8d98-10247346980e.png)
![Screenshot (851)](https://user-images.githubusercontent.com/2363959/214562554-03f8365b-b57f-4ddc-924e-b4b29c82968b.png)

**Comment by jongough** [comment]
What is the issue? If OD us not installed and active then all this plugin can do is initialise. 

It is designed to test the ODAPI binary and JavaScript interfaces. It is also a minimalistic plugin that could be used as the basis for a new plugin as it has all the basics needed to work with OCPN.

**Comment by rgleason** [comment]
Good point. We need Ocpn_draw installed in O5.7.1 to have it work I guess.
The screens are out of scale for me.

**Comment by jongough** [comment]
The OD issue is wx32 related and the changes needed for icon scaling. I will get to that when I have a development machine.

**Comment by rgleason** [comment]
I believe this has been fixed in 1.0.226  on rgleason repository https://github.com/rgleason/testplugin_pi/commits/master
